probe_src/libprobe/src/fd_table.c

```diff
@@ -1,3 +1,53 @@
+#include <pthread.h>
+#include <assert.h>
```

Comment:
Let's just remove this file; it is no longer included in the build.

Reply:
both of them right?